src/Microsoft.Diagnostics.Runtime/Native/NativeRuntime.cs

```diff
@@ -261,6 +275,95 @@ private void InitThreads()
             _threads = threads.ToArray();
         }
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
+        {
+            if (!this.ClrInfo.DacInfo.PlatformAgnosticFileName.Contains("mrt100"))
+                throw new ClrDiagnosticsException("Serialized exceptions are only available for a Native data target");
+
+            if (_sosNativeSerializedExceptionSupport == null)
+                throw new ClrDiagnosticsException("This version of mrt100 is too old.", ClrDiagnosticsException.HR.DataRequestError);
+
+            var exceptionById = new Dictionary<ulong, NativeHeap.NativeException>();
+            ISerializedExceptionEnumerator serializedExceptionEnumerator = _sosNativeSerializedExceptionSupport.GetSerializedExceptions();
+            while (serializedExceptionEnumerator.HasNext())
+            {
+                ISerializedException serializedException = serializedExceptionEnumerator.Next();
+
+                //build the stack frames
+                IList<ClrStackFrame> stackFrames = new List<ClrStackFrame>();
+                ISerializedStackFrameEnumerator serializedStackFrameEnumerator = serializedException.StackFrames;
+                while (serializedStackFrameEnumerator.HasNext())
+                {
+                    ISerializedStackFrame serializedStackFrame = serializedStackFrameEnumerator.Next();
+
+                    NativeModule nm = ((NativeHeap)this.GetHeap()).GetModuleFromAddress(serializedStackFrame.IP);
+                    if (nm == null)
+                    {
+                        Trace.WriteLine(String.Format("Unable to resolve module for IP {0}", serializedStackFrame.IP));
+                        continue;
+                    }
+                    if (nm.Pdb == null) {
+                        Trace.WriteLine(String.Format("PDB not found for IP {0}, Module={1}", serializedStackFrame.IP,nm.Name));
+                        continue;
+                    }
+
+                    ISymbolResolver resolver = null;
+                    try
+                    {
+                        resolver = this.DataTarget.SymbolProvider
+                                        .GetSymbolResolver(nm.Pdb.FileName, nm.Pdb.Guid, nm.Pdb.Revision);
+                    }
+                    catch (Exception e)
+                    {
+                        Trace.WriteLine($"Error in finding the symbol resolver for PDB [Filename:{nm.Pdb.FileName}, GUID:{nm.Pdb.Guid}, Revision:{nm.Pdb.Revision}]: {e.Message}");
+                        Trace.WriteLine("Check previous traces for additional information");
+                        continue;
```

Comment:
Continuing is incorrect.  That will lead to a stack trace with missing frames when we can't resolve the name.  Instead `string symbolName` should simply be null when we cannot resolve the name.  The stack trace will contain the IP (but not its resolved name) and the consumer of the API can try again to resolve that IP themselves.